asset-transfer-private-data/chaincode-go/README.md

```diff
@@ -230,22 +174,22 @@ Error: endorsement failure during invoke. response: status:500 message:"appraisa
 
 Nor is a member of Org2 able to read the Org1 private data collection:
 ```
-peer chaincode query -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n private_transfer -c '{"function":"ReadAssetPrivateDetails","Args":["Org1MSPPrivateCollection","asset1"]}'
+peer chaincode query -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n private -c '{"function":"ReadAssetPrivateDetails","Args":["Org1MSPPrivateCollection","asset1"]}'
 ```
 By setting `"memberOnlyRead": true` in the collection configuration file, we specify that only members of of Org1 can read data from the collection. A member who tries to read the collection would only get the following response.
```

Comment:
question: 

the collection config have `"memberOnlyWrite": false,`
which seem to suggest that non-members can Write to the collection. Can it be set to true to avoid confusion?

Reply:
"memberOnlyWrite": false, is actually part of the sample. A member of the other organization cannot read the value of another organization, but they can write to it during a transaction. Remember, the organization would need to endorse that write, so malicious behavior would be prevented.